index.html

```diff
@@ -144,6 +144,25 @@ <h3>About me</h3>
         </ul>
 
     </section>
+
+    <footer class="contact-me">
+        <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        <form action="https://formspree.io/f/mayvwpkl" method="post">
+
+            <input type="text" name="fullname" id="name" placeholder="Full name" maxlength="30" required> 
+            <input type="email" name="email-address" id="email" placeholder="Email address" required>
+            <textarea name="message" id="message" maxlength="500" required>Enter text here</textarea>
```

Comment:
- [ ] [OPTIONAL] Kindly accept my suggestion. In here `Enter text here` should be a place holder.


```suggestion
            <textarea name="message" id="message" maxlength="500" placeholder="Enter text here" required></textarea>
```